L1Trigger/TrackFindingTracklet/src/HitPatternHelper.cc

```diff
@@ -21,13 +21,13 @@ namespace hph {
         oldKFPSet_(iConfig.getParameter<edm::ParameterSet>("oldKFPSet")),
         setupTT_(setupTT),
         dataFormats_(dataFormats),
-        dfcot_(dataFormats_.format(trackerTFP::Variable::cot, trackerTFP::Process::kfin)),
-        dfzT_(dataFormats_.format(trackerTFP::Variable::zT, trackerTFP::Process::kfin)),
+        dfcot_(dataFormats_.format(trackerTFP::Variable::cot, trackerTFP::Process::gp)),
```

Comment:
Is the HitPatternHelper, which is only being used in the Hybrid (or Tracklet) algo,  being made dependent on the GP, which is only used in the TMTT algo. What is going on here>

Reply:
KF depends on GP, Hybrid uses KF.